index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Instruments</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>Music</header>
+      <a href="#main_types_of_instruments" class="nav-link" id="main-types">Main Types of Instruments</a>
+      <a href="#wind_instruments" class="nav-link">Wind Instruments</a>
+      <a href="#string_instruments" class="nav-link">String Instruments</a>
+      <a href="#percution_instruments" class="nav-link">Percution Instruments</a>
+      <a href="#electronic_instruments" class="nav-link">Electronic Instruments</a>
```

Comment:
- [ ] I think it would be better if you use a `<ul>` tag with each `<a>` element inside of the `<li>` tags for better accessibility and better semantic practice. You can later use Flexbox to improve your design.